### PR TITLE
GH action - auto-update PRs

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

This shall update PR which is ready-to-go but need to be tested in action as not sure how it will act if there are some collisions, and if it works we can also drop a comment message
see: https://github.com/marketplace/actions/auto-update

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by SeanNaren — APPROVED** [review]
Nice this is awesome. Hopefully works out!

**Comment by awaelchli** [comment]
"update" branch means it will trigger CI too? that would be very costly considering drone takes almost 20 mins per run

**Comment by Borda** [comment]
> "update" branch means it will trigger CI too? that would be very costly considering drone takes almost 20 mins per run

it will work only with `ready-to-go` label, as @SeanNaren asked for :]

**Review by awaelchli — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Review by s-rog — APPROVED** [review]
Didn't know this existed!

File: .github/workflows/events-recurrent.yml [hunk 1]
```diff
@@ -13,9 +13,25 @@ jobs:
...
+  # see: https://github.com/marketplace/actions/auto-update
+  pr-auto-update:
+    name: Auto-update PR
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: docker://chinthakagodawita/autoupdate-action:v1
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          DRY_RUN: "false"
+          PR_FILTER: "labelled"
```
Comment (tchaton) [inline .github/workflows/events-recurrent.yml:33]: Shouldn't you filter `has conflicts` ?

**Review by tchaton — APPROVED** [review]

File: .github/workflows/events-recurrent.yml (hunk 1 above)
Comment (justusschock) [reply .github/workflows/events-recurrent.yml:33]: This does not seem to be available for this. In the mentioned action it states that this is coming soon...

**Review by justusschock — APPROVED** [review]

File: .github/workflows/events-recurrent.yml (hunk 1 above)
Comment (Borda) [reply .github/workflows/events-recurrent.yml:33]: good point, adding it here #5480

**Pull request merged**